### PR TITLE
Update cosmos kit packages

### PR DESCRIPTION
**Pull request by NoahSaso**
## What is the purpose of the change

This updates Cosmos Kit packages to make a handful of UX improvements:
- Fix a fee bug introduced in a recent version of Cosmos Kit. The workaround is removed in this PR since the fix was implemented.
- Fix Keplr Mobile deep linking support.
- Add a simple placeholer image for the iframe wallet when the outer wallet is disconnected.
- Added recursive architecture to the iframes so that nested apps can use their own iframes internally.

## Testing and Verifying

This change has been tested locally. Test it by trying to submit a transaction and watching it succeed without having to manually set a fee in Keplr.

### CODE REVIEW

**Comment by NoahSaso** [comment]
@JoseRFelix this undoes a fix from the past PR (#2252) by updating Cosmos Kit which just patched it.

**Review by JoseRFelix — APPROVED** [review]
Thank you, LGTM!

**Comment by NoahSaso** [comment]
snuck another cosmos kit version update in that adds a placeholder image for the iframe wallet (UX improvement)

File: packages/stores/package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@cosmjs/math": "0.29.0",
     "@cosmjs/proto-signing": "0.29.0",
     "@cosmjs/stargate": "0.29.0",
-    "@cosmos-kit/core": "2.7.0",
```
Comment (JoseRFelix) [inline packages/stores/package.json:56]: This version should match `web/package.json`

**Review by JoseRFelix — COMMENTED** [review]
Thank you for this PR! Left a small comment

File: packages/stores/package.json (hunk 1 above)
Comment (NoahSaso) [reply packages/stores/package.json:56]: oops great catch

**Comment by NoahSaso** [comment]
> Thank you for this PR! Left a small comment

nice thank you! fixed

**Pull request merged**